grails-app/services/org/pih/warehouse/data/DataService.groovy

```diff
@@ -702,12 +702,14 @@ class DataService {
                 } else if (element.defaultValue && !value) {
                     value = element.defaultValue
                 }
-                properties[fieldName] = value ?: ""
+                // We can't just check the truthiness of the value, because the false boolean would be evaluated to an empty string
+                properties[fieldName] = value == null ? "" : value
```

Comment:
can't we just use elvis operator here?

Reply:
@alannadolny no we can't, I explained it in the comment above, that we couldn't just check the truthiness of the value, because the `false` boolean would be evaluated to an empty string, which we want to avoid.